app/models/utils.rb

```diff
@@ -25,6 +27,15 @@ def self.parse_param(params, id, parse_method)
     end
   end
 
+  ##
+  # Raises ValidationError unless given hash contains specified key
+  def self.raise_error_unless_has_key(hash, key, message, key_prefix='')
```

Comment:
Wstawienie tej metody do utils jest dla mnie trochę kontrowersyjne, ale rozumiem, że to przez to, żeby nie modyfikować gema.